projekt_do_pracy_na_labach/fiszki_app/docker_compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+  backend:
+    build:
+      context: ./fiszki_backend
+    ports:
+      - "5000:5000"
+    depends_on:
+      - db
+    networks:
+      - app-network
+    environment:
+      DATABASE_URL: "mongodb://db:27017"
+
+  frontend:
+    build:
+      context: ./fiszki_app
+    ports:
+      - "8000:8000"
```

Comment:
Tutaj jak zmienisz port udostępniany z kontenera na 80 to będzie działać. (Rozwiązanie podaję bo nie dodałem w dockerfile'u frontendu klauzuli EXPOSE). TLDR aplikacja jest odpalana z NGinx'a w kontenerze który domyślnie działa na porcie `80`